### ISSUE TITLE
Deprecated

### ISSUE THREAD

**Issue by cdalexndr** [kenwheeler/slick#4068]
Last release is in 2017.
Shouldn't this repository be deprecated due to inactivity? (like putting a warning in readme)

**Comment by lorvent** [comment]
it is sad to see a plugin like this being end up abondoned.

I wish someone takes it up and moves it forward.

**Comment by cdalexndr** [comment]
I guess an alternative that is actively developed would be [Swiper](https://github.com/nolimits4web/swiper).

**Comment by lorvent** [comment]
@cdalexndr yes swiper is being maintained actively but swiper is much more powerful and heavy.

where as slick is really slick, in small sites where i don't need much effects....i feel slick works great.
